### PR TITLE
DG - Write-Up on `workout /new` Feature

### PR DESCRIPTION
**Pull request by alanlowzies**
This PR contains the following additions/changes:
- DG: Write-up on `workout /new` feature
- UG: Update some of the `workout /list` examples
- UG: Update file structure information

### CODE REVIEW

**Review by a1021492980 — APPROVED** [review]
Looks good, thank you:)

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -87,10 +87,13 @@ You are now ready to begin developing!
 * [Getting User Input Continuously](#getting-user-input-continuously)
 * [Parsing User Input and Getting the Right Command](#parsing-user-input-and-getting-the-right-command)
 * [Create New Workout](#create-new-workout)
+    * [Design Considerations](#design-considerations-for-creating-a-new-workout) 
 * [Search](#search)
   * [Search for Exercise](#search-for-exercise)
   * [Search for Plan](#search-for-plan)
```
[the remark this answers is not in the document]
Comment (alanlowzies) [reply docs/DeveloperGuide.md:93]: > Good and clear navigation!

Thank you :)

**Review by Musfirahe0556596 — APPROVED** [review]
Very detailed. Team may want to consider writing like this to follow a certain format ^_^

**Comment by alanlowzies** [comment]
> Very detailed. Team may want to consider writing like this to follow a certain format ^_^

Thank you 🙂 🙂 🙂

**Pull request merged**